chain/types/blockheader.go

```diff
@@ -199,32 +199,6 @@ var blocksPerEpoch = NewInt(build.BlocksPerEpoch)
 
```

Comment:
the `blocksPerEpoch` above this line can be moved to electionproof.go where it is now used